punchcut/src/outline.rs

```diff
@@ -0,0 +1,293 @@
+/*!
+Representation of a simple outline glyph.
+*/
+
+use peniko::kurbo::{
+    segments, BezPath, ParamCurveArclen, ParamCurveArea, ParamCurveExtrema, PathEl, Point, Rect,
+    Shape,
+};
+
+use crate::source::*;
+
+/// Simple outline represented as a sequence of path commands.
+#[derive(Clone, Default, Debug)]
+pub struct Outline {
+    verbs: Vec<Verb>,
+    points: Vec<(f32, f32)>,
```

Comment:
Just for my edification, what's the rationale for using this representation, instead of an enum ala `kurbo::PathEl`? Is this about better memory efficiency?

Reply:
Right. I think I made the rationale in the above comment about f32 in kurbo. This is probably moot now though.